qa/rpc-tests/p2p-acceptblock.py

```diff
@@ -271,8 +291,82 @@ def run_test(self):
 
         test_node.sync_with_ping()
         assert_equal(self.nodes[0].getblockcount(), 1442)
+        self.nodes[0].getblock(all_blocks[1438].hash)
+        assert_equal(self.nodes[0].getbestblockhash(), all_blocks[1438].hash)
+        assert_raises_jsonrpc(-1, "Block not found on disk", self.nodes[0].getblock, all_blocks[1439].hash)
         print("Successfully reorged to longer chain from non-whitelisted peer")
 
+        # 8. Create a chain which is invalid at a height longer than the
+        # current chain, but which has more blocks on top of that
+        block_1441f = create_block(all_blocks[1436].sha256, create_coinbase(1441), all_blocks[1436].nTime+1)
+        block_1441f.solve()
+        block_1442f = create_block(block_1441f.sha256, create_coinbase(1440), block_1441f.nTime+1)
+        block_1442f.solve()
+        block_1443 = create_block(block_1442f.sha256, create_coinbase(1441), block_1442f.nTime+1)
+        # block_1443 spends a coinbase below maturity!
+        block_1443.vtx.append(create_transaction(block_1442f.vtx[0], 0, b"42", 1))
+        block_1443.hashMerkleRoot = block_1443.calc_merkle_root()
+        block_1443.solve()
+        block_1442 = create_block(block_1443.sha256, create_coinbase(1444), block_1443.nTime+1)
+        block_1442.solve()
```

Comment:
```suggestion
        block_1444 = create_block(block_1443.sha256, create_coinbase(1444), block_1443.nTime+1)
        block_1444.solve()
```
I think the block number in the variable name is wrong

Reply:
https://github.com/bitcoin/bitcoin/blob/0.16/test/functional/p2p_unrequested_blocks.py#L250

Reply:
https://github.com/dogecoin/dogecoin/blob/cfb47206b8d548db2e2197f5c625a91107158a18/qa/rpc-tests/p2p-acceptblock.py#L300

Reply:
agreed oops 🙏♥️🥂

Reply:
@xanimo All good - it's clear how it ended up with that number, but as far as I can tell it's never appended at 1,442.

Reply:
> @xanimo All good - it's clear how it ended up with that number, but as far as I can tell it's never appended at 1,442.

no you're absolutely correct was sleepy mistake. should absolutely 1444. unfortunately my internet just went out (on phone) so if when patrick sees this perhaps he can correct my blunder.